web/client/reducers/geostory.js

```diff
@@ -218,7 +256,11 @@ export default (state = INITIAL_STATE, action) => {
         }
         return set(path, newElement, state);
     }
+    case UPDATE_SETTINGS: {
+        return set(`currentStory.settings.${action.prop}`, action.value, state);
+    }
     case UPDATE_CURRENT_PAGE: {
+        /** if column update state only if the column for the current immersive section has changed */
```

Comment:
```suggestion
        /* if the page update updates a column, update state only if the column for the current immersive section has changed.
        * This to avoid to select a column that doesn't belong to the current section (it could happen due to scroll events, in case of multiple immersive sections in the same viewport).
        */
```
This is a comment, not documentation